salt/modules/cmdmod.py

```diff
@@ -2727,6 +2751,15 @@ def powershell(cmd,
 
     :param str saltenv: The salt environment to use. Default is 'base'
 
+    :param int depth: The number of levels of contained objects to be included.
+        Default is 2. Values greater than 4 seem to greatly increase the time
+        it takes for the command to complete for some commands. eg: ``dir``
+
+        .. versionadded:: 2016.3.4
+
+    :returns:
+        :dict: A dictionary of data returned by the powershell command.
```

Comment:
Again, wrong. Not a dictionary. It would be accurate to say "A JSON-formatted string of data returned by the powershell command."